.github/workflows/android_beta_deployment.yml

```diff
@@ -0,0 +1,101 @@
+name: Android Beta Deployment
+
+on:
+  push:
+    branches:
+    - 'feature/**'
```

Comment:
1. Do we want to build on every feature?
2. Maybe build only if there is a PR?

Reply:
Currently I’m using feature branches only for testing